mozilla_version/gecko.py

```diff
@@ -565,6 +663,23 @@ class FennecVersion(_VersionWithEdgeCases):
         'build_number': 1,
     })
 
+    _RELEASED_EDGE_CASES_MAJOR = ({
+        'major_number': 1,
+        'minor_number': 1,
+    }, {
+        'major_number': 14,
+        'minor_number': 0,
+        'patch_number': 1,
```

Comment:
For an unknown reason, the current product details doesn't any track record of what build number we had on Fennec 14.0.1. That's the reason why it's missing here.